config/kafka-monitor.properties

```diff
@@ -39,6 +39,9 @@
     "zookeeper.connect": "localhost:2181",
     "bootstrap.servers": "localhost:9092",
     "produce.record.delay.ms": 100,
+    "produce.topic.autoTopicCreationEnabled": true,
```

Comment:
If the defaults are changed in kafka-monitor.properties then why do they need to be changed in BasicEndToEndTest.java?  Are these not the properties for the basic end to end test?

Yes, things should work as advertised.  I will make the other changes.